docs/cheatsheet.rst

```diff
@@ -63,7 +63,7 @@ Global Variables
   with arbitrary precision and does not wrap around at ``2**256``. Assert that ``k != 0`` starting from version 0.5.0.
 - ``this`` (current contract's type): the current contract, explicitly convertible to ``address`` or ``address payable``
 - ``super``: the contract one level higher in the inheritance hierarchy
-- ``selfdestruct(address payable recipient)``: destroy the current contract, sending its funds to the given address
+-  ``selfdestruct(address payable recipient)``: (**Deprecated**) destroy the current contract and transfer its remaining funds to the specified recipient address (deprecated starting from Solidity 0.8.0, use transfer or send instead)
```

Comment:
Please, remove extra whitespace. Also, I'm not sure if we need to change the wording more than something like: `(**Deprecated since 0.8.0**)` or something like that.

Reply:
I would even argue that it can be removed from the cheatsheet completely, so as not to encourage its use.

Reply:
Oh, my bad! Let me fix all of it and make a push in a bit, thought different PRs for different files would make for an easier time in tracking, but this is def more cumbersome. 

I'd argue we should keep it but use the deprecated tag as suggested